python/tvm/relay/op/_transform.py

```diff
@@ -1,8 +1,51 @@
 #pylint: disable=invalid-name, unused-argument
 """Backend compiler related feature registration"""
 from __future__ import absolute_import
+import topi
+import topi.cuda
+from tvm import container
 from . import op as _reg
-from .op import schedule_injective, OpPattern
+from .op import (schedule_injective, register_compute, register_schedule,
+                 register_pattern, OpPattern)
+
+schedule_broadcast = schedule_injective
+
+# squeeze
+@register_compute("squeeze")
+def squeeze_compiler(attrs, inputs, output_type, target):
+    """Compiler for squeeze dims."""
```

Comment:
squeeze already support negative index, and None I think so you just have to redirect to topi call

Reply:
I think we should rename this to "compute" to match the style in the rest of the system.